src/Raven.Client/Documents/Operations/ETL/SQL/SqlProviderParser.cs

```diff
@@ -8,7 +8,7 @@ public static SqlProvider GetSupportedProvider(string factoryName)
         {
             switch (factoryName)
             {
-                case "System.Data.SqlClient":
+                case "Microsoft.Data.SqlClient":
```

Comment:
We must not change that. Existing connection strings will have `"System.Data.SqlClient"`. We should just add `"Microsoft.Data.SqlClient"`